metrics/metrics.go

```diff
@@ -0,0 +1,220 @@
+// Copyright 2018 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package metrics
+
+import "github.com/prometheus/client_golang/prometheus"
+
+// TiKVClient metrics.
+var (
+	TiKVTxnCounter = prometheus.NewCounter(
+		prometheus.CounterOpts{
+			Namespace: "tidb",
```

Comment:
seem it is better to use tikv as the namespace and client as the subsystem now. But we need to update our Grafana too